common.gypi

```diff
@@ -407,9 +407,15 @@
         'libraries': [ '-lelf' ],
       }],
       ['OS=="freebsd"', {
-        # Use this flag because on FreeBSD std::pairs copy constructor is non-trivial
-        # https://lists.freebsd.org/pipermail/freebsd-toolchain/2016-March/002094.html
-        'cflags': [ '-D_LIBCPP_TRIVIAL_PAIR_COPY_CTOR=1' ],
+        'conditions': [
+          ['llvm_version < "4.0"', {
```

Comment:
Good quastion, but in that case `get_llvm_version` would fail since it uses this RegEx `r"(^(?:FreeBSD )?clang version|based on LLVM) ([3-9]\.[0-9]+)")`
https://github.com/nodejs/node/blob/master/configure#L585